src/include/ceph_features.h

```diff
@@ -70,6 +70,7 @@
 #define CEPH_FEATURE_OSD_HITSET_GMT (1ULL<<54)
 #define CEPH_FEATURE_HAMMER_0_94_4 (1ULL<<55)
 #define CEPH_FEATURE_NEW_OSDOP_ENCODING   (1ULL<<56) /* New, v7 encoding */
+#define CEPH_FEATURE_NEW_OSDREPOP_ENCODING   (1ULL<<57) /* New, v2 encoding */
```

Comment:
@liewegas Is it possible to make this feature combined with 56? I guess it should be timely